10000shotapp/10000_shots
# created all migrations

**What's this PR do?**
- add all migrations files

**Where should the reviewer start?**
- check migration files

**How should this be manually tested?**
- run migration using php artisan migrate command and also check rollback is working perfectly or not.
migration command : php artisan migrate
rollback command : php artisan migrate:rollback


**Any background context you want to provide?**
- N/A
